lib/geocoder/tomtomgeocoder.js

```diff
@@ -31,19 +30,23 @@ TomTomGeocoder.prototype._geocode = function(value, callback) {
   var _this = this;
 
   var params = {
-    query : value,
-    key   : this.options.apiKey,
-    format: 'json'
+    key   : this.options.apiKey
   };
 
-  this.httpAdapter.get(this._endpoint, params, function(err, result) {
+  if (this.options.language) {
+    params.language = this.options.language;
+  }
+
+  var url = this._endpoint + '/' + value + '.json';
```

Comment:
do you need to escape value?  maybe use `encodeURIComponent` here  
<img width="892" alt="Screen Shot 2019-05-27 at 3 12 56 PM" src="https://user-images.githubusercontent.com/1336873/58436156-ec06e980-8091-11e9-9f3c-f6085ecca589.png">

Reply:
As it did work for me, it might be that it did get encoded somewhere else? Maybe inside the httpAdapter?